server/api/index.js

```diff
@@ -1,31 +1,31 @@
-const Shopify = require('shopify-api-node');
-const express = require('express');
-const axios = require('axios');
+const Shopify = require("shopify-api-node");
+const express = require("express");
+const axios = require("axios");
```

Comment:
When I save it changes to a single cote, I think something related to my vs version